Src/AutoFixture.xUnit.net/InfiniteDataAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit.Extensions;
+
+namespace Ploeh.AutoFixture.Xunit
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class InfiniteDataAttribute : DataAttribute
```

Comment:
This can actually be an idea worth considering. – May I suggest to discuss this separately (e.g. as a new issue)?